### DISCUSSION TITLE
菜单国际化

### DISCUSSION THREAD

**Discussion by Ivan-YangSj** [ant-design/ant-design-pro#11218] [Q&A]
菜单是怎么实现国际化的，我配置了config的locale、layout开启，locales中也添加了相应的"menu.xxx"，路由的name也与xxx匹配，但是国际化没生效

**Comment by Fettes** [comment]
https://umijs.org/docs/max/layout-menu#locale
找这个不对吗？ 
config下的layout下的配置
<img width="610" alt="image" src="https://github.com/ant-design/ant-design-pro/assets/16219214/1489fafe-3508-4938-a89a-4092c018b60d">
在这个里面，别搞错地方了
